lib/Browser.js

```diff
@@ -122,21 +122,27 @@ class Browser extends EventEmitter {
     return version.product;
   }
 
+  _disconnected() { this.emit(Browser.Events.Disconnected, 'connection disconnected');}
```

Comment:
nit: formatting here is improper. The "inlining" is to remove the function `_disconnected` and just emit the event in `disconnected` method right away.

Reply:
done